roles/openshift_node/tasks/main.yml

```diff
@@ -0,0 +1,41 @@
+---
+
+# tasks file for openshift_node
+- name: Install OpenShift
+  yum: pkg=openshift-node state=installed
+
+  # fixme: Once openshift stops resolving hostnames for node queries remove this...
+- name: Set hostname to IP Addr (WORKAROUND)
+  command: /usr/bin/hostname {{ oo_bind_ip }}
```

Comment:
@akram I'm changing the hostname to be the IP address to satisfy kubernetes expectations, you are just setting a variable yes?